src/tsconfig-base.json

```diff
@@ -4,8 +4,8 @@
         "outDir": "../built/local",
 
         "pretty": true,
-        "lib": ["es2020"],
-        "target": "es2020",
+        "lib": ["es2021"],
+        "target": "es2021",

```

Comment:
This bump is just to add `WeakRef` to the lib (which is supported in `node14`+). I mention it in a comment above - this can work without weak refs, it'd just guarantee transient symbols always stick around forever, rather than maybe getting collected if they weren't actually kept around anywhere. But a transient symbol not being stores somewhere more checker-lifetime'd is probably rare, so it's *probably* OK to just not use weak refs to track transient symbols if there's an issue with that. It's more defensive than anything else.